src/preprocessing/preprocessing_pass_registry.cpp

```diff
@@ -0,0 +1,46 @@
+/*********************                                                        */
+/*! \file preprocessing_pass_registry.cpp
+ ** \verbatim
+ ** Top contributors (to current version):
+ **  Justin Xu
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2017 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief Preprocessing pass registry
+ **
+ ** Preprocessing pass registry that uses a name to store instances of
+ ** classes in a map. Makes sure passes are properly disposed of when
+ ** registry is destroyed.  
+ **/
+
+#include "preprocessing/preprocessing_pass_registry.h"
+
+#include <utility>
+
+#include "base/cvc4_assert.h" 
+#include "base/output.h"
+#include "preprocessing/preprocessing_pass.h"
+
+namespace CVC4 {
+namespace preprocessing {
+
+void PreprocessingPassRegistry::registerPass(
+    const std::string& ppName, std::unique_ptr<PreprocessingPass> preprocessingPass) {
+  Debug("pp-registry") << "Registering pass " << ppName << std::endl;
+  Assert(preprocessingPass != NULL);
```

Comment:
nullptr

Reply:
Done, also changed another occurrence of `NULL` in preprocessing_pass.cpp.